prometheus/process_collector_test.go

```diff
@@ -101,3 +104,65 @@ func TestProcessCollector(t *testing.T) {
 		t.Errorf("%d metrics collected, want 1", n)
 	}
 }
+
+// TestNewPidFileFn ...
```

Comment:
Final nit: Test functions don't require a doc comment. If you want to provide a meaningful doc comment, go for it. But this one is not meaningful. I'd just delete this line.